paddle/phi/api/yaml/op_compat.yaml

```
@@ -1091,30 +1091,60 @@
       support_tensor : true
 
 - op : fake_channel_wise_quantize_abs_max
+  inputs :
+    {x : X, bit_length : bit_length, round_type : round_type, quant_axis : quant_axis, is_test : is_test}
```

```suggestion
    {x : X}
```
这里如果名称相同的话，就不用配置映射了。下面也需要一起删除一下